archetypes/liferay-portlet-spring-mvc-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+
+<archetype-descriptor name="liferay-portlet-spring-mvc-archetype">
+	<fileSets>
+		<fileSet filtered="true">
+			<directory>src/main/java</directory>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/webapp</directory>
+			<excludes>
+				<exclude>icon.png</exclude>
+			</excludes>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/webapp/jsp</directory>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/webapp/jsp/spring-context/portlet</directory>
+			<includes>
```

Comment:
Hey @jtydhr88  here you mention including some files:  but those are in the archetype refernces.  Those files do  exist here: https://github.com/jtydhr88/liferay-plugins/tree/2d0eabe1d707bdc6cc642afa5e7f157e1e884ee4/tools/templates/portlet_spring_mvc_tmpl/docroot/WEB-INF/spring-context